src/adapter/src/catalog/open.rs

```diff
@@ -254,7 +254,7 @@ impl Catalog {
 
         let mut storage = config.storage;
         let is_read_only = storage.is_read_only();
-        tracing::debug!("Catalog::open - beginning initial transaction.");
+        tracing::warn!("Catalog::open - beginning initial transaction.");
```

Comment:
I thought that `warn` had a negative implication, this seems like the expected code path to take.

Reply:
The tracing crate says this about the warn level:

> Designates hazardous situations.

https://docs.rs/tracing/latest/tracing/struct.Level.html#associatedconstant.WARN

Reply:
Good catch, reduced it to `info`!